app/models/rescan_runner.rb

```diff
@@ -136,7 +136,7 @@ def process_file(location, codec, path)
 
       track_artists = track_artists.map { |ta| TrackArtist.new(artist: ta[:artist], name: ta[:name], role: ta[:role], order: ta[:order]) }
 
-      genre = (Genre.find_by(name: t_genre) || Genre.new(name: t_genre) if t_genre.present?)
+      genre = (Genre.find_by(normalized_name: t_genre.normalize) || Genre.new(name: t_genre) if t_genre.present?)
```

Comment:
```suggestion
      genre = (Genre.find_by(normalized_name: Genre.normalize(t_genre)) || Genre.new(name: t_genre) if t_genre.present?)
```
(I removed the String monkey-patch when reworking the PR, since I am very much against monkey-patching unless absolutely necessary.)